src/commands/Minion/pickpocket.ts

```diff
@@ -65,7 +65,10 @@ export default class extends BotCommand {
 			quantity = null;
 		}
 
-		const pickpocketable = Pickpocketables.find(npc => stringMatches(npc.name, name));
+		await msg.author.settings.sync(true);
```

Comment:
why did you add this?

Reply:
Apologies, copied it over accidentally and everything worked fine in testing so didn't pick up on it